lib/whitehall/edition_batch_link_checker.rb

```diff
@@ -0,0 +1,30 @@
+# Calls the Link Checker API to verify all links in an edition per organisation
+module Whitehall
+  class EditionBatchLinkChecker
```

Comment:
This class feels like a worker to me, is there a reason it shouldn't be? It would mean we can then create a rake task to run this task manually, and also get Sidekiq to run it every 24 hours.